### PR TITLE
ci(dep): introduce groups feature in dependabot

### PR DESCRIPTION
**Pull request by clearloop**
Ref #3345 

dependabot has introduced grouped feature in 2023-06-30 the https://github.blog/changelog/2023-06-30-grouped-version-updates-for-dependabot-public-beta/

## Open Question

How to test it...

@gear-tech/dev

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -2,27 +2,39 @@ version: 2
 updates:
   - package-ecosystem: "cargo"
     directory: "/"
-    labels: ["A0-pleasereview", "A4-insubstantial", "E2-forcemacos"]
+    labels: ["A4-insubstantial", "E2-forcemacos"]
     schedule:
       interval: "weekly"
       day: "friday"
```
Comment (breathx) [inline .github/dependabot.yml:8]: and consider moving it on sat-sun mb to avoid hanging ci on Friday

**Review by breathx — APPROVED** [review]

**Comment by breathx** [comment]
Can we provide on workflow dispatch as well to test it?

**Review by shamilsan — APPROVED** [review]
Agree with @breathx about the `/examples/` entry. We don't need it anymore.

File: .github/dependabot.yml (hunk 1 above)
Comment (clearloop) [reply .github/dependabot.yml:8]: moved to `Sunday` so that we can have a chill Saturday and handle it on Monday

**Pull request merged**